src/components/layout/custom/Generator.jsx

```diff
@@ -63,7 +63,7 @@ export default function Generator({
               key={i}
               item
               {...Utility.getSizes(subBlock.gridSizes)}
-              style={subBlock.gridStyle || { textAlign: 'center' }}
+              style={subBlock.style || { textAlign: 'center' }}
```

Comment:
Hmm I'm not sure about this one. I think this might be a breaking change, I thought the `.style` prop customize the component inside of the Grid?